services/agent/src/simcore_service_agent/core/application.py

```diff
@@ -41,7 +41,7 @@ def _setup_logger(settings: ApplicationSettings):
 def create_app() -> FastAPI:
     settings = ApplicationSettings.create_from_envs()
     _setup_logger(settings)
-    logger.debug(settings.model_dump_json(indent=2))
+    logger.debug(settings.model_dump_json(indent=2, warnings="none"))
```

Comment:
not sure I fully understand why the warning pops up. does this not mean the settings object fields contain something not correct?

Reply:
I've debugged it up to this point. When `settings` is serialized to JSON using `model_dump_json`, it triggers this warning.  

I'm not sure why it complains about `settings` having an `Enum` field serialized as a string, given that it's a validated `Enum` field.  

That said, since this serialization is only for printing, I ignore all warnings there.  

Still, I agree this isn't fully resolved, and we need to move forward with it.

Reply:
@sanderegg I reverted the `warning='none'` everwhere. The problem was in the logger validator.